Symfony/CS/Tests/Fixer/Contrib/AlignDoubleArrowFixerTest.php

```diff
@@ -30,6 +30,77 @@ public function provideFixCases()
     {
         return array(
             array(
+                '$array = array(
+                "closure" => function ($param1, $param2) {
+                    return;
+                }
+            );', ),
+            array(
+                "<?php
+        return new JsonResponse(array(
+            'result' => 'OK',
+            'html'   => 1, array(
+                'foo'    => 'bar',
+                'foofoo' => array(
+                    'a'  => 1,
+                    'b'  => 2
+                )
+            ),)
+        );",
+                "<?php
+        return new JsonResponse(array(
+            'result' => 'OK',
+            'html' => 1, array(
+                'foo' => 'bar',
+                'foofoo' => array(
+                    'a' => 1,
+                    'b'  =>  2
+                )
+            ),)
+        );",
+                ),
+            array(
+                "<?php
+        return new JsonResponse([
+            'result' => 'OK',
+            'html'   => renderView('views/my_view.html.twig', array(
+                'foo'    => 'bar',
+                'foofoo' => 42,
+            )),
+        ]);",
+                "<?php
+        return new JsonResponse([
+            'result' => 'OK',
+            'html' =>    renderView('views/my_view.html.twig', array(
+                'foo' => 'bar',
+                'foofoo' => 42,
+            )),
+        ]);",
+
```

Comment:
I think it is, since it is the only test for long array with a short array in it starting on the same line.